package.json

```diff
@@ -1,15 +1,22 @@
 {
   "name": "amcharts3-angular2",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
Please don't bump the version. I prefer to group multiple changes into a single version.